viz/DepthMapVisualization.cpp

```diff
@@ -98,10 +100,15 @@ void DepthMapVisualization::updateMainNode ( osg::Node* node )
     }
     else if(show_remission && !scan_sample.remissions.empty())
     {
-        for(unsigned i = 0; i < scan_sample.remissions.size(); i++)
+        for(unsigned i = 0; i < points.size(); i++)
         {
 	    if(scan_sample.isIndexValid(i))
```

Comment:
If you change the index from remissions to points, you probably should validate that points.size() == remissions.size().